h2o-assembly/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
   compile project(":h2o-app")
   compile project(":h2o-persist-s3")
   compile project(":h2o-persist-hdfs")
+  compile project(':h2o-scala_2.10')
```

Comment:
yup, I know. We also need to support artefacts for scala_2.11, so this is not a permanent solution, I just did it in order to be able to test it.

However this wouldn't be necessary if the class-loading in h2o would work as expected.

When I tried to start h2o: `java -cp h2o-scala.jar:h2o.jar clazz` it worked but when h2o needed to load class H2OFrame it still ended on a error that it doesn't know the class H2OFrame defined in h2o-scala.

So maybe fixing this is good starting point.